app/scripts/modules/kubernetes/loadBalancer/configure/wizard/basicSettings.html

```diff
@@ -0,0 +1,63 @@
+<form class="container-fluid form-horizontal" name="form" novalidate>
+  <div ng-if="!state.accountsLoaded" style="height: 200px">
+    <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
+  </div>
+  <div class="modal-body" ng-if="state.accountsLoaded">
+    <div class="form-group">
+      <div class="col-md-12 well" ng-class="{'alert-danger': form.loadBalancerName.$error.validateUnique, 'alert-info': !form.loadBalancerName.$error.validateUnique}">
+        <strong>Your load balancer will be named:</strong>
+        <span>{{ctrl.getName()}}</span>
+          <!-- Angular does not seem to run length validation on hidden inputs, hence the text + display:none -->
+        <input type="text"
+               style="display: none"
+               ng-maxlength="253"
+               class="form-control input-sm"
+               ng-model="loadBalancer.name"
+               validate-unique="existingLoadBalancerNames"
+               validate-ignore-case="true"
+               name="loadBalancerName"/>
+        <validation-error ng-if="form.loadBalancerName.$error.validateUnique" message="There is already a load balancer in {{loadBalancer.credentials}} with that name."></validation-error>
+      </div>
+    </div>
+    <div class="form-group">
+      <div class="col-md-3 sm-label-right">Account</div>
+      <div class="col-md-7">
+        <account-select-field component="loadBalancer" field="credentials" accounts="accounts" provider="'kubernetes'" on-change="ctrl.accountUpdated()"></account-select-field>
+      </div>
+    </div>
+      <namespace-select-field component="loadBalancer" field="namespace" account="loadBalancer.credentials" namespaces="namespaces" on-change="ctrl.namespaceUpdated()"></namespace-select-field>
+    <div class="form-group">
+      <div class="col-md-3 sm-label-right">Stack <help-field key="aws.loadBalancer.stack"></help-field></div>
```

Comment:
Should probably rename that help key from `aws.loadBalancer.stack` at some point.